express/scripts/scripts.js

```diff
@@ -128,6 +129,10 @@ const miloLibs = setLibs(LIBS);
   document.head.append(footerMeta);
 
   listenMiloEvents();
+  if (getMetadata('sheet-powered') === 'Y') {
+    autoUpdateContent(document.getElementsByTagName('main')[0], miloLibs)
```

Comment:
you should check for `area` instead of `('main')[0]` so that your code works with fragments too.